presto-mixin/dashboards/presto-coordinator-overview.libsonnet

```diff
@@ -0,0 +1,1343 @@
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 15,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'ms',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [
+        'min',
+        'max',
+        'mean',
+      ],
+      displayMode: 'table',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local cpuTimeConsumedOneMinuteRatePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'com_facebook_presto_execution_QueryManager_ConsumedCpuTimeSecs_OneMinute_Rate{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} ',
+      format='time_series',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'CPU time consumed - one minute rate',
+  description: 'The rate at which queries are consuming CPU time.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 15,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: '/ sec',
```

Comment:
This panel, description, and units are really difficult to understand. Can you please update the description and use a more appropriate rate unit for this?

Reply:
I altered this panel a bit to make it more clear.
1. Clarified on the description
2. Changed the query from a OneMinute_Rate -> OneMinute_Count. Since the metric itself originally returns seconds, a rate on top of that would make it sec/sec or / sec which can be a bit confusing. By changing this into the OneMinute_Count i can just keep a flat sec as the unit while making the panel overall make a bit more sense.
Let me know what you think

Reply:
Yep that works, thank you for explaining :)